lib/ae_mdw/db/util.ex

```diff
@@ -239,11 +260,11 @@ defmodule AeMdw.Db.Util do
 
   @spec gen_to_txi(Blocks.height()) :: Txs.txi()
   def gen_to_txi(gen) do
-    case Database.fetch(Model.Block, {gen, -1}) do
```

Comment:
Changed not needed anymore (same result). This will be reverted in a following PR.